cli/cmd/sync.go

```diff
@@ -39,6 +42,7 @@ func NewCmdSync() *cobra.Command {
 	cmd.Flags().String("license", "", "set offline license file")
 	cmd.Flags().String("summary-location", "", "Sync summary file location. This feature is in Preview. Please provide feedback to help us improve it.")
 	cmd.Flags().String("tables-metrics-location", "", "Tables metrics file location. This feature is in Preview. Please provide feedback to help us improve it. Works with plugins released on 2024-07-10 or later.")
+	cmd.Flags().String("shard", "", "Allows splitting the sync process into multiple shards. This feature is in Preview. Please provide feedback to help us improve it. Works with plugins released on 2024-09-24 or later.")
```

Comment:
This does not mention that it only works for plugins using the default scheduler (e.g. not S3), but I realise it's a little tricky to put this in words.

Reply:
Yeah, it doesn't work with Docker plugins as well 🙃 

I think that will be in the update of https://docs.cloudquery.io/docs/advanced-topics/running-cloudquery-in-parallel and https://docs.cloudquery.io/docs/deployment/github-actions

Reply:
OK so did https://github.com/cloudquery/cloudquery/pull/19169/commits/fff4d1d79a46f1eeff5702d1ca8f14970fe58fff